simulation/sub8_gazebo/worlds/gripper.world

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+  <sdf version="1.4">
```

Comment:
I used it to test the gripper so I didn't have to reload all the sub stuff to make small tweaks